src/sentry/south_migrations/0371_auto__add_servicehook.py

```diff
@@ -0,0 +1,1058 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
+
+    def forwards(self, orm):
+        # Adding model 'ServiceHook'
+        db.create_table('sentry_servicehook', (
+            ('id', self.gf('sentry.db.models.fields.bounded.BoundedBigAutoField')(primary_key=True)),
+            ('guid', self.gf('django.db.models.fields.CharField')(max_length=32, unique=True, null=True)),
+            ('application', self.gf('sentry.db.models.fields.foreignkey.FlexibleForeignKey')(to=orm['sentry.ApiApplication'], null=True)),
+            ('actor_id', self.gf('sentry.db.models.fields.bounded.BoundedPositiveIntegerField')(db_index=True)),
+            ('project_id', self.gf('sentry.db.models.fields.bounded.BoundedPositiveIntegerField')(db_index=True)),
+            ('url', self.gf('django.db.models.fields.URLField')(max_length=200)),
+            ('secret', self.gf('sentry.db.models.fields.encrypted.EncryptedTextField')(default='4ad44bc577044c8db84185cff62095f35ad4cf2aafe8442b89c8042e5468b0e8')),
```

Comment:
Just noting this because this is... weird. Not sure if it'll be a problem. But this value is the default that came from `ServiceHook.generate_secret()` which is unencrypted. Not that the encryption is a problem here, but in the database, the value should be encrypted. This just means we're setting a database level default which can never be read back. :) I wish there was a way to set `default` without it translating to a database level default. Since these don't make sense when `default` is a python callable.

Reply:
Fixed -- agreed this is super annoying and easy to look over